internal/app/machined/internal/sequencer/v1alpha1/types.go

```diff
@@ -62,6 +62,10 @@ func (d *Sequencer) Boot() error {
 			"mount extra devices",
 			userdatatask.NewExtraDevicesTask(),
 		),
+		phase.NewPhase(
```

Comment:
This no longer needs to happen because of #1205. The PKI is generated when OSD starts which is the last phase.